### DISCUSSION TITLE
Can BOM files be hybrid

### DISCUSSION THREAD

**Discussion by omerfinger** [CycloneDX/specification#338] [Q&A] [answered]
Hi, is it common to create a BOM file that contains both components (SBOM) and vulnerabilities (VEX) sections? 
If so, what type of a BOM is it?

**Comment by stevespringett** [comment, marked as answer]
It is common to have a BOM that contains vulnerability information (VDR) as many security tools, especially SCA, produce such output.  See https://cyclonedx.org/capabilities/vdr/

Optionally the analysis section can be done as well, but is much less prevalent. See https://cyclonedx.org/capabilities/vex/
